nixos/modules/services/databases/postgresql.nix

```diff
@@ -439,7 +439,7 @@ in
             ]);
             options = {
               shared_preload_libraries = mkOption {
-                type = nullOr (coercedTo (listOf str) (concatStringsSep ", ") str);
+                type = nullOr (coercedTo (listOf str) (concatStringsSep ",") commas);
```

Comment:
I tried changing the type but didn't remove the extra space. Let me test that and get back to you!